qa/L0_http/generate_models/mock_llm/config.pbtxt

```diff
@@ -41,6 +41,12 @@ input [
     name: "STREAM"
     data_type: TYPE_BOOL
     dims: [ 1, 1 ]
+  },
+  {
+    name: "input_ids"
```

Comment:
Do we need a new input to test this? Or can we validate the same behavior (support input with list of size > 1 IIUC), by sending multiple prompts? ex: `"PROMPT": ["hello", "world"]` with updated `dims: [1, -1]`